README.md

```diff
@@ -11,6 +11,51 @@ The backend for a generic mobile-friendly quiz engine created using FastAPI and
 
 ### Local DB Setup
 
+
+- The following steps are for a Linux.
```

Comment:
for a linux what?

Also, would suggest to not keep this section heading as a bullet point, but a `h4` heading.
Like `#### Linux Systems`

And same for mac section
`### Mac Systems`